originals/t.yaml

```diff
@@ -54,6 +54,23 @@
   - OSX
   - Windows
 
+- name: Teeworlds
```

Comment:
OSGC doesn't list original open source games, so teeworlds should not be added

Reply:
for original open source games consider contributing to projects like https://trilarion.github.io/opensourcegames

Reply:
What about DDraceNetwork? I was considering to add it here too, it's a teeworlds fork that is being distributed as a standalone Game on Steam

Reply:
it shouldn't be added either imo

Reply:
Ok